scopes/docs/docs/overview/overview.tsx

```diff
@@ -21,8 +22,10 @@ export function Overview({ titleBadges }: OverviewProps) {
   const componentDescriptor = useComponentDescriptor()
   const lanesModel = useLanesContext();
   const currentLane = lanesModel?.currentLane;
-  const { data } = useFetchDocs(component.id.toString());
-  const fetchComponent = data?.component;
+
+  const envType: string = componentDescriptor?.get<any>('teambit.envs/envs')?.type;
```

Comment:
can you get this from envs aspect id? or at least extract to const at the top of the file